learn/on-premise/observability/metrics-collection/alerts.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Alerts"
+id: ""
+sidebar_label: "Alerts"
+---
+---
+
+WaveMaker uses the **Prometheus AlertManager** to listen to the data accumulated at *Prometheus* and fire an alert based on the query included in the alert.
+
+### Accessing Alerts
+
+The list of Alerts can be accessed from Prometheus UI by clicking on the "**Alerts**" button as shown in the below image
+
+![Prometheus UI Alert Button](/learn/assets/wme-setup/wme-observability/prometheus/prometheus-alert-button-click.png)
+
+### Available Alerts
+
+Following are the list of alerts included in WaveMaker
+
```

Comment:
These alerts table should be divided into separate tables. 

Alerts on key end user features
Alerts on internal system health